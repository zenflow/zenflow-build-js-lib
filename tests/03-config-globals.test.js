/* eslint-env jest */

const path = require('path')
const buildFixture = require('./util/buildFixture')
const { readFile } = require('./util/fs')

describe('config-globals', () => {
  test('snapshot built umd', async () => {
    const tempDir = await buildFixture('config-globals')
    const file = path.join(tempDir, 'dist/config-globals-fixture.umd.js')
    expect(await readFile(file)).toMatchSnapshot()
  })
})
