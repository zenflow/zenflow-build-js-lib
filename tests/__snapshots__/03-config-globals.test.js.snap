// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config-globals snapshot built umd 1`] = `
"/** @preserve
  * package: config-globals-fixture v0.0.0
  * file: dist/config-globals-fixture.umd.js
  * homepage: HOMEPAGE
  * license: LICENSE
  **/
(function (global, factory) {
	typeof exports === 'object' && typeof module !== 'undefined' ? module.exports = factory(require('const-sqrt-half'), require('const-sqrt-two')) :
	typeof define === 'function' && define.amd ? define(['const-sqrt-half', 'const-sqrt-two'], factory) :
	(global.configGlobalsFixture = factory(global.constSqrtHalf,global.customConstSqrtTwo));
}(this, (function (SQRT_HALF,SQRT2) { 'use strict';

	SQRT_HALF = SQRT_HALF && SQRT_HALF.hasOwnProperty('default') ? SQRT_HALF['default'] : SQRT_HALF;
	SQRT2 = SQRT2 && SQRT2.hasOwnProperty('default') ? SQRT2['default'] : SQRT2;

	var index = SQRT_HALF + SQRT2;

	return index;

})));
"
`;
